test/sys/test_pthread.rs

```diff
@@ -20,3 +20,16 @@ fn test_pthread_kill_none() {
     pthread_kill(pthread_self(), None)
         .expect("Should be able to send signal to my thread.");
 }
+
+#[test]
+#[cfg(all(target_os = "linux", not(target_env = "musl")))]
+fn test_pthread_name_np() {
+    let tid = pthread_self();
+    let name = b"nix-rust";
+    let namestr = std::str::from_utf8(name).unwrap();
+    pthread_setname_np(tid, name.to_vec()).expect("Should pass");
```

Comment:
`expect` should only be used in combination with a useful message.  If you're just going to say "Should pass", then you might as well just use `unwrap()`.